README.md

```diff
@@ -1,70 +1,38 @@
-### Docker users, see [Moby and Docker](https://mobyproject.org/#moby-and-docker) to clarify the relationship between the projects
-
-### Docker maintainers and contributors, see [Transitioning to Moby](#transitioning-to-moby) for more details
-
 The Moby Project
 ================
 
 ![Moby Project logo](docs/static_files/moby-project-logo.png "The Moby Project")
 
-Moby is an open-source project created by Docker to advance the software containerization movement.
-It provides a “Lego set” of dozens of components, the framework for assembling them into custom container-based systems, and a place for all container enthusiasts to experiment and exchange ideas.
```

Comment:
I liked the concept of "lego set" 😢

Reply:
@thaJeztah wasnt sure if we should be using trademarks like that... but it still is!

Reply:
Yes, I do like it, but wasn't sure if we were allowed to use it (without mentioning trademarks in a footnote 😅)

Reply:
Apparently there will be Lego at DockerCon EU though!

Reply:
I think this sentence falls pretty clearly under "fair use". I propose we re-open a PR to bring it back.

Reply:
👍  for bring back "Lego set". Added a PR #35156.

Reply:
merged that, thanks